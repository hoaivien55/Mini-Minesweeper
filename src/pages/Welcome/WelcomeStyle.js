import styled from "styled-components";
import { Container, Row } from "../../components/styles";

const WelcomeContainerStyle = styled(Container)`
	box-shadow: 0px 5px 10px 5px rgb(255 178 199 / 20%);
    border-radius: 12px;
    border: 1px solid rgb(255 178 199 / 40%);
`;

const WelComeHeaderStyle = styled(Row)`
	position: relative;
    margin-bottom: 20px;
	&::after {
		content: " ";
		width: 120px;
		height: 100px;
		background-image: url(/boom.png);
		position: absolute;
		top: -110%;
		right: -17%;
		z-index: 1000;
		background-position: center;
		background-size: contain;
		background-repeat: no-repeat;
	}
`;

export { WelcomeContainerStyle, WelComeHeaderStyle };
