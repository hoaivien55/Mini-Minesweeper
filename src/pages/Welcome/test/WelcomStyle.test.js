import renderer from "react-test-renderer";
import { WelcomeContainerStyle, WelComeHeaderStyle } from "../WelcomeStyle";

it("WelcomeContainerStyle: Virtual DOM snapshot", () => {
	const tree = renderer.create(<WelcomeContainerStyle />).toJSON();
	expect(tree).toMatchSnapshot();
});

it("WelComeHeaderStyle: Virtual DOM snapshot", () => {
	const tree = renderer.create(<WelComeHeaderStyle />).toJSON();
	expect(tree).toMatchSnapshot();
});
