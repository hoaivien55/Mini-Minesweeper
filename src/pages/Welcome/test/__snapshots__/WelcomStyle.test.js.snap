// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelComeHeaderStyle: Virtual DOM snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0px;
}

.c1 {
  position: relative;
  margin-bottom: 20px;
}

.c1::after {
  content: " ";
  width: 120px;
  height: 100px;
  background-image: url(/boom.png);
  position: absolute;
  top: -110%;
  right: -17%;
  z-index: 1000;
  background-position: center;
  background-size: contain;
  background-repeat: no-repeat;
}

<div
  className="c0 c1"
/>
`;

exports[`WelcomeContainerStyle: Virtual DOM snapshot 1`] = `
.c0 {
  position: absolute;
  left: 50%;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  margin: 0 auto;
  width: auto;
  min-width: 300px;
  min-height: 200px;
  padding: 20px;
  background-color: white;
}

.c1 {
  box-shadow: 0px 5px 10px 5px rgb(255 178 199 / 20%);
  border-radius: 12px;
  border: 1px solid rgb(255 178 199 / 40%);
}

<div
  className="c0 c1"
/>
`;
